scripts/update-tezos.sh

```diff
@@ -8,14 +8,14 @@
 
 set -e
 
-git config user.name "Savely Krendelhoff" # necessary for pushing
-git config user.email "alicecandyom@proton.me"
+git config user.name "serokell-bot" # necessary for pushing
+git config user.email "tezos-packaging@serokell.io"
```

Comment:
I see now that the changes in this file have been reverted in a later commit (0acb266efc1fe1cd220a95f674dbed7659679782), but they shouldn't be committed at all IMO